src/Microsoft.OData.Core/ODataConstants.cs

```diff
@@ -144,6 +144,9 @@ internal static class ODataInternalConstants
         /// <summary>A segment name in a URI that indicates metadata is being requested.</summary>
         internal const string UriMetadataSegment = "$metadata";
 
+        /// <summary>The OData prefix</summary>
+        internal const string ODataPrefix = "odata";

```

Comment:
>odata [](start = 45, length = 5)

JsonLightConstants defines "ODataAnnotationNamespacePrefix = "odata.";

Can we re-use it?

Reply:
I would rather not have uriparser reference jsonlightconstants. Perhaps we should just move it.

---
In reply to: [166082858](https://github.com/OData/odata.net/pull/1063#discussion_r166082858) [](ancestors = 166082858)